### DISCUSSION TITLE
Unchecking Checkbox does not toggle variable to false

### DISCUSSION THREAD

**Discussion by Kaleidosium** [alpinejs/alpine#3182] [1. Help] [answered]
Example Code can be found here: https://codepen.io/kaleidosium/pen/QWrmZxO?editors=1112, unfortunately I couldn't figure out how to make it run properly on Codepen, it's really only for the code.

This GIF should hopefully explain it well, basically the checkbox enables a flag that increases the price, however, it does not reduce the price when I uncheck it, I checked in console and it seems like the flag doesn't toggle back to `false` on unchecking the box.

![checkboxbug](https://user-images.githubusercontent.com/39834015/193281222-ef7064cd-cb66-4f9b-9e2e-835d1202ef63.gif)

**Comment by ekwoka** [comment]
Your codepen is completely broken so I can't really check the reproducibility of this.

I can say, the code looks quite a bit messed up. Like I'm not sure the use of the `fieldset` here...just model the value directly on the input, and also use a `getter` for the price instead of a funciton you have to call.

**Reply by Kaleidosium** [reply]
I'm using [7.css](https://github.com/khang-nd/7.css), which requires them for this.

**Reply by ekwoka** [reply]
Okay, just put the model on the input itself then.

**Reply by Kaleidosium** [reply]
Alright, I got codepen to work. https://codepen.io/kaleidosium/pen/QWrmZxO

**Reply by ekwoka** [reply]
<img width="931" alt="Screen Shot 2022-09-30 at 6 33 45 PM" src="https://user-images.githubusercontent.com/43540491/193293334-cf3c6c67-72e1-42c1-a5f8-18fa58c963c1.png">

Still broken.

**Reply by Kaleidosium** [reply]
I think it _should_ work now.
![image](https://user-images.githubusercontent.com/39834015/193295015-a1f4a296-c98d-411a-8552-2f5ac72e8f5a.png)

**Comment by Kaleidosium** [comment, marked as answer]
Alright, with the help of @ekwoka in a discord server, you can find the updated codepen here: https://codepen.io/kaleidosium/pen/QWrmZxO, thanks so much!
